daemon/api_test.go

```diff
@@ -288,6 +286,8 @@ func (s *apiBaseSuite) daemon(c *check.C) *Daemon {
 		Serial: "serialserial",
 	})
 
+	snapstate.CanAutoRefresh = nil
```

Comment:
Why was this moved? Seems much safer to have it very early on while the suite is being setup rather than on a function after much logic has been run, and even the daemon was created.

Reply:
because the overlord manager's init now writes that variable

Reply:
Can we please have a comment? Seems tempting to move it elsewhere.